src/cljs/reviews_next/pages/login.cljs

```diff
@@ -1,6 +1,7 @@
 (ns reviews-next.pages.login)
-
+ 
 (defn login []
   [:div.container
-   [:div {:class "g-signin2"
+   [:div
+     {:class "g-signin2"
```

Comment:
This kind of spacing is not usual in clojure code. 
You can look at https://github.com/bbatsov/clojure-style-guide

We also have a Dev-dependency on cljfmt, an automatic code formatter for clojure. You can setup your editor to do the formatting on file save. https://marketplace.visualstudio.com/items?itemName=pedrorgirardi.vscode-cljfmt

This avoids [bike-shedding](https://en.wiktionary.org/wiki/bikeshedding) in a project.